Modules/Scripted/DICOMPlugins/DICOMScalarVolumePlugin.py

```diff
@@ -531,17 +532,38 @@ def setVolumeNodeProperties(self, volumeNode, loadable):
             #   [2] https://github.com/Slicer/Slicer/blob/3bfa2fc2b310d41c09b7a9e8f8f6c4f43d3bd1e2/Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.h#L172
             #
             try:
-                windowCenter = float(slicer.dicomDatabase.fileValue(file, self.tags["windowCenter"]))
-                windowWidth = float(slicer.dicomDatabase.fileValue(file, self.tags["windowWidth"]))
                 displayNode = volumeNode.GetDisplayNode()
                 if displayNode:
-                    logging.info("Window/level found in DICOM tags (center=" + str(windowCenter) + ", width=" + str(windowWidth) + ") has been applied to volume " + volumeNode.GetName())
-                    displayNode.AddWindowLevelPreset(windowWidth, windowCenter)
+                    # Retrieve the window center and window width values from the DICOM file
+                    windowCenterValues = slicer.dicomDatabase.fileValue(file, self.tags["windowCenter"])
+                    windowWidthValues = slicer.dicomDatabase.fileValue(file, self.tags["windowWidth"])
+
+                    # Split the values by the backslash character to handle multiple values
+                    windowCenterList = windowCenterValues.split("\\")
+                    windowWidthList = windowWidthValues.split("\\")
+
+                    # Ensure both lists have the same length
+                    if len(windowCenterList) != len(windowWidthList):
+                        raise ValueError("The number of window center values does not match the number of window width values")
+
+                    if len(windowCenterList) == 0:
+                        logging.info("No window/level found in DICOM tags. SOPinstanceUID: " + uid)
```

Comment:
Reduce noise a bit and consistently report both volume node name and SOP instance UID.
```suggestion
                        logging.debug(f"No window/level found for volume {volumeNode.GetName()} in DICOM tags of SOP instance {uid}.")
```